### PR TITLE
Added solution to 08

### PR DESCRIPTION
**Pull request by RubelOS**
Using Seaborn, mathplotlib, numpy

### CODE REVIEW

File: Exercises/JuliusDralle/Exercise_08.ipynb [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+   "id": "d98bf38f-e9f3-46bf-9cb6-783f8dfeb585",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    },
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('Data/titanic.csv')\n",
```
[the remark this answers is not in the document]
Comment (RubelOS) [reply Exercises/JuliusDralle/Exercise_08.ipynb:70]: Yes, totally understandable. I am using Jupyter Lab and my code is not required to be used by others which is why i am not using relative paths or pathlib

**Review by joh-lin — APPROVED** [review]
Comments have been added.

File: Exercises/JuliusDralle/Exercise_08.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (LeMixer) [reply Exercises/JuliusDralle/Exercise_08.ipynb:66]: Well, reviewers want to review your code though without having to write a couple lines just to get yours going.

**Comment by LeMixer** [comment]
You've added a couple comments. The solution looks good now. Feel free to merge.

**Review by LeMixer — APPROVED** [review]
You've added a couple comments. The solution looks good now. Feel free to merge.

**Comment by fabischw** [comment]
I will be merging this PR if it does not get merged by the PR creator within the next 3 days in an effort to clean up the repo (More information can be found in [Issue #228](https://github.com/dhbw-ma-ppp/ppp-2023/issues/228))

**Pull request merged**